src/Features/Core/Portable/CodeRefactorings/IRefactoringHelpersService.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.CodeRefactorings
+{
+    interface IRefactoringHelpersService : ILanguageService
+    {
+        /// <summary>
```

Comment:
Should we just move the doc comment to the service? Otherwise, please add doc comments to all overloads.

Reply:
Will rework docs for everything.